manifests/v1beta1/installs/katib-with-kubeflow/kustomization.yaml

```diff
@@ -2,51 +2,32 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: kubeflow
 resources:
-  # Katib controller.
-  - ../../components/controller/controller.yaml
-  - ../../components/controller/service.yaml
-  - ../../components/controller/rbac.yaml
-  - ../../components/controller/katib-config.yaml
-  - ../../components/controller/trial-templates.yaml
-  # Katib CRDs.
-  - ../../components/crd/experiment.yaml
-  - ../../components/crd/suggestion.yaml
-  - ../../components/crd/trial.yaml
-  # Katib DB manager.
-  - ../../components/db-manager/db-manager.yaml
-  - ../../components/db-manager/service.yaml
-  # Katib DB mysql.
-  - ../../components/mysql/mysql.yaml
-  - ../../components/mysql/service.yaml
-  - ../../components/mysql/pvc.yaml
-  - ../../components/mysql/secret.yaml
-  # Katib UI.
-  - ../../components/ui/ui.yaml
-  - ../../components/ui/service.yaml
-  - ../../components/ui/rbac.yaml
-  # Katib webhooks.
-  - ../../components/webhook/cert-generator.yaml
-  - ../../components/webhook/rbac.yaml
-  - ../../components/webhook/webhooks.yaml
+  - ../katib-cert-manager
   # Kubeflow Katib components.
   - kubeflow-katib-roles.yaml
   - ui-virtual-service.yaml
 images:
   - name: docker.io/kubeflowkatib/katib-controller
-    newTag: latest
     newName: docker.io/kubeflowkatib/katib-controller
-  - name: docker.io/kubeflowkatib/katib-db-manager
     newTag: latest
+  - name: docker.io/kubeflowkatib/katib-db-manager
     newName: docker.io/kubeflowkatib/katib-db-manager
-  - name: docker.io/kubeflowkatib/katib-ui
     newTag: latest
+  - name: docker.io/kubeflowkatib/katib-ui
     newName: docker.io/kubeflowkatib/katib-ui
-  - name: docker.io/kubeflowkatib/cert-generator
     newTag: latest
+  - name: docker.io/kubeflowkatib/cert-generator
```

Comment:
I think we can remove this image since we are not using cert-generator image in the Katib with Kubeflow install.

Reply:
Good catch! Just pushed a commit with these removed.